app/models/site.rb

```diff
@@ -3,6 +3,10 @@ class Site < ActiveRecord::Base
 
   validates :application_name, presence: true, allow_nil: true
 
+  # Allow for uploading of site's banner image
+  mount_uploader :banner_image, BannerImageUploader, mount_on: :banner_image_name
```

Comment:
Can we rename the `banner_image_name` column back to `banner_image` and just use Carrierwave's defaults?